docs/src/app/(public)/(content)/react/handbook/animation/page.mdx

```diff
@@ -38,7 +38,7 @@ For example, if the user closes a popup before it finishes opening, with CSS tra
 Use the following Base UI attributes for creating CSS animations when a component becomes visible or hidden:
 
 - `[data-open]` corresponds to the style applied when a component becomes visible.
-- `[data-ending-style]` corresponds the style applied before a component becomes hidden.
+- `[data-closed]` corresponds the style applied before a component becomes hidden.
```

Comment:
Technically `[data-open]` isn't necessary, and `[data-ending-style]` _can_ be used, but isn't used in the code block below. This feels nice symmetrically, though.